.github/workflows/ci.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
             deps
             _build/test
             priv/plts
-          key: ${{ runner.os }}-${{ env.OTP_VERSION }}-${{ env.ELIXIR_VERSION }}-${{ hashFiles('mix.lock') }}
+          key: ${{ runner.os }}-${{ env.OTP_VERSION }}-${{ env.ELIXIR_VERSION }}-${{ hashFiles('mix.lock') }}-${GITHUB_REF##*/}
```

Comment:
Can we split this into another PR?

Reply:
This PR would fail :sweat_smile: 
Maybe we can merge the _another PR_ first

What do you prefer?

Reply:
I'd prefer to split and merge the other PR, but I don't really get why we're getting into this behavior ._.

Reply:
I could observe is that the [linked broken run](https://github.com/trento-project/wanda/runs/7989424887?check_suite_focus=true) fails with a `{:error, :schema_not_found}`
```
1) test serializing outgoing events should produce an ExecutionCompletedV1 with a valid event data, as per the schema (Wanda.MapperTest)
     test/messaging/mapper_test.exs:31
     match (=) failed
     code:  assert :ok = Wanda.JsonSchema.validate(event_type, event_data)
     left:  :ok
     right: {:error, :schema_not_found}
     stacktrace:
       test/messaging/mapper_test.exs:40: (test)
```
meaning it cannot find the file `priv/json_schema/trento.checks.v1.ExecutionCompleted.schema.json` which is actually there committed and pushed.

By triggering a cache rebuild the pipeline is happy again.

This is at least _what_ happens. I am not sure about the _why_: it looks like what's in `priv/json_schema/` gets bundled in the deps cache. 
No idea :sweat_smile:

Reply:
> I'd prefer to split and merge the other PR, but I don't really get why we're getting into this behavior ._.

Here you go https://github.com/trento-project/wanda/pull/12